app/src/main/java/com/hover/stax/ApplicationInstance.kt

```diff
@@ -80,6 +81,10 @@ class ApplicationInstance : Application() {
         AppsFlyerLib.getInstance().init(getString(R.string.appsflyer_key), conversionListener, this)
     }
 
+    companion object {
```

Comment:
I don't love this, in general try to avoid adding to applicationinstance. 

@alexosugo what do you think, is there a better approach? My first thought is to use shared prefs

Reply:
Merging for now, will open a chore on pivotal tracker to make this better